python/src/opendp/context.py

```diff
@@ -262,6 +261,9 @@ def range_warning(name, value, info_level, warn_level):
         U = RuntimeType.parse_or_infer(U, rho)
         return zero_concentrated_divergence(T=U), rho
 
+    if epsilon is None:
+        raise ValueError("Either epsilon or rho must be specified.")
```

Comment:
Won't this raise if I call it like `loss_of(rho=.5)`?

Reply:
Unless I'm confused, no: If `rho` is set we'll return zero_concentrated_divergence just above.